### ISSUE TITLE
This repository does not work at all (Printer not connected after connect to usb device)

### ISSUE THREAD

**Issue by jooskesters** [bebo925/ng-thermal-print#42]
I have tested this on windows and linux. Both result in the same scenario. The console says printer not connected. After clicking connect to my printer in the popup it again says printer not connected.

**Comment by andalike** [comment]
Works for me, Have you tried to call the requestUsb function?

**Comment by benwood162** [comment]
I'd like to second this, if anyone could help out? 
after calling the requestUsb function and selecting the device from the chrome popup, in the console log i get the device details followed by "not connected to printer"

**Comment by andalike** [comment]
> I'd like to second this, if anyone could help out?
> after calling the requestUsb function and selecting the device from the chrome popup, in the console log i get the device details followed by "not connected to printer"

Is this a Windows machine? I had faced the issue when connecting to the Windows Machine.

**Comment by benwood162** [comment]
yep.. did some further digging in other issues and found the answer here! 
#https://github.com/bebo925/ng-thermal-print/issues/11#issuecomment-607634549

**Comment by andalike** [comment]
> yep.. did some further digging in other issues and found the answer here!
> #[#11 (comment)](https://github.com/bebo925/ng-thermal-print/issues/11#issuecomment-607634549)

In case it helps, You can try this(will save your time)
1) Please install the Zadigs Software and RESTART the Computer
2) More on this on:
    .) https://github.com/WICG/webusb/issues/184
    ..)https://stackoverflow.com/questions/49078873/windows-10-webusb-printing-issues-with-zebra-lp2844-domexception-access-denied
3) This might not work, So "Remove Let Windows manage my printer" and also "Remove Printers which are added to the Windows Printer"
4) Then Map the "Windows Service" in "Zadigs software"
